src/ucs/config/parser.c

```diff
@@ -1424,7 +1424,7 @@ void ucs_config_parse_config_files()
     /* Library dir */
     path_p = ucs_sys_get_lib_path();
     if (path_p != NULL) {
-        strncpy(path, path_p, PATH_MAX - 1);
+        ucs_strncpy_safe(path, path_p, PATH_MAX - 1);
```

Comment:
instead of PATH_MAX - 1 can use PATH_MAX or sizeof(path) since ucs_strncpy_safe includes the NULL terminator